config/config.exs

```diff
@@ -33,6 +33,12 @@ config :ueberauth, Ueberauth.Strategy.Github.OAuth,
   client_id: System.get_env("GITHUB_CLIENT_ID"),
   client_secret: System.get_env("GITHUB_CLIENT_SECRET")
 
+config :rummage_ecto, Rummage.Ecto,
+  default_repo: Tudo.Repo,
+  default_per_page: 20,
```

Comment:
do we need both `default_per_page ` and `max_page`?

Reply:
No sorry max_page is now doing nothing. I'll fix it.